examples/tutorials/grpc/src/main/proto/blog.proto

```diff
@@ -0,0 +1,30 @@
+syntax = "proto3";
+
+package example.armeria.server.blog.grpc;
+option java_package = "example.armeria.server.blog.grpc";
+
+import "google/protobuf/timestamp.proto";
+
+service BlogService {
+  rpc CreateBlogPost (BlogPost) returns (BlogPost) {}
+  rpc GetBlogPost (BlogId) returns (BlogPost) {}
+  rpc GetBlogPosts (Sort) returns (stream BlogPost) {}
+  rpc UpdateBlogPost (BlogPost) returns (BlogPost) {}
+  rpc DeleteBlogPost (BlogId) returns (BlogId) {}
```

Comment:
How about following the API guideline?
```suggestion
  rpc CreateBlogPost (CreateBlogPostRequest) returns (BlogPost) {}
  rpc GetBlogPost (GetBlogPostRequest) returns (BlogPost) {}
  rpc ListBlogPosts (ListBlogPostsRequests) returns (ListBlogPostsResponse) {}
  rpc UpdateBlogPost (UpdateBlogPostRequest) returns (BlogPost) {}
  rpc DeleteBlogPost (DeleteBlogIdRequest) returns (google.protobuf.Empty) {}
```
https://google.aip.dev/133
https://google.aip.dev/135
...

Reply:
Actually, I tried to follow the method names of annotated services in the blog tutorial.
But seems good to abide by google.api.dev